bodo/pandas/utils.py

```diff
@@ -951,3 +962,29 @@ def count_plan(self):
 
     data = execute_plan(projection_plan)
     return data[0]
+
+
+def ensure_datetime64ns(df):
+    """Convert datetime columns in a DataFrame to 'datetime64[ns]' dtype.
+    Avoids datetime64[us] that is commonly used in Pandas but not supported in Bodo.
```

Comment:
The docs and the code don't match.  The code avoids everything except ns right?  I recall some other code above where we support second, milli, micro, nano so I think we need to be consistent and define which precisions we support and if it is only us we avoid then the code should reflect that.

Reply:
We convert non-ns precisions to ns in some places but I agree we should clarify and be more consistent. Generally, anything that goes through Arrow is fine but Pandas support with Numpy dtypes may have issues.